symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/impl/AbstractDatafeedLoop.java

```diff
@@ -77,7 +77,7 @@ public void unsubscribe(RealTimeEventListener listener) {
    */
   @Override
   public void start() throws AuthUnauthorizedException, ApiException {
-    if (this.started.get()) {
+    if (!this.started.compareAndSet(false, true)) {
```

Comment:
Datafeed start flag must be check and set with an atomic operation as very first command of start/stop APIs (or both API must be synchronized, but I preferred to avoid this, also for using VirtualThread on client side for registering listeners).
Otherwise, during the start command, started flag was set only inside runLoop method, but I take some time to arrive on it (before, there is datafeed initialization) and we can have concurrency issue stopping it